### ISSUE TITLE
Error when trying to run with new Firefox Developer Edition 70.0b2 (64-bit)

### ISSUE THREAD

**Issue by icl7126** [mozilla/web-ext#1692]
### Is this a feature request or a bug?
BUG

### What is the current behavior?
1) install Firefox Developer Edition
2) using any project, execute:
```
web-ext run --firefox=firefoxdeveloperedition

Running web extension from C:\Users\Juraj\git\addons\GroupSpeedDial\firefox_dev
Use --verbose or open Tools > Web Developer > Browser Console to see logging

o: Temporary add-on installation is not supported in this version of Firefox (you need Firefox 49 or higher). For older Firefox versions, use --pre-install
    at c.sourceDir [as startFirefoxInstance] (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\dist\webpack:\src\extension-runners\firefox-desktop.js:274:51)
    at processTicksAndRejections (internal/process/task_queues.js:85:5)

[firefox-profile] cannot delete profileDir on exit C:\Users\Juraj\AppData\Local\Temp\aa93c460-53dc-4bca-9c14-2ff1221e2121 Error: EBUSY: resource busy or locked, unlink 'C:\Users\Juraj\AppData\Local\Temp\aa93c460-53dc-4bca-9c14-2ff1221e2121\cert9.db'
    at Object.unlinkSync (fs.js:1018:3)
    at rimrafSync (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\fs-extra\lib\remove\rimraf.js:254:15)
    at C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\fs-extra\lib\remove\rimraf.js:291:39
    at Array.forEach (<anonymous>)
    at rmkidsSync (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\fs-extra\lib\remove\rimraf.js:291:26)
    at rmdirSync (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\fs-extra\lib\remove\rimraf.js:281:7)
    at Object.rimrafSync (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\fs-extra\lib\remove\rimraf.js:252:7)
    at FirefoxProfile._cleanOnExit (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\firefox-profile\lib\firefox_profile.js:253:10)
    at process.FirefoxProfile.onExit (C:\Users\Juraj\AppData\Roaming\npm\node_modules\web-ext\node_modules\firefox-profile\lib\firefox_profile.js:144:12)
    at process.emit (events.js:214:15) {
  errno: -4082,
  syscall: 'unlink',
  code: 'EBUSY',
  path: 'C:\\Users\\Juraj\\AppData\\Local\\Temp\\aa93c460-53dc-4bca-9c14-2ff1221e2121\\cert9.db'

```
### Version information (for bug reports)

Firefox Developer Edition 70.0b2 (64-bit)
Win 10 1903 x64
* Paste the output of these commands:

````
node --version && npm --version && web-ext --version
v12.9.0
6.10.1
3.1.1
````

EDIT:  
Same problem with current Nightly. 
Beta 69 (currently release candidate), works fine.

**Comment by Rob--W** [comment]
Are you able to manually remove the supposedly locked file?

To see if this is a Firefox issue, could you run [`mozregression`](https://mozilla.github.io/mozregression/) ? You can use the following command to run `web-ext` with `mozregression`:

```
mozregression -g 69 -b 70 -c 'web-ext run -f {binary}'
```

**Comment by icl7126** [comment]
The file is locked only while the newly opened Firefox stays opened (the script fails, but the gets opened anyway).

I will try the regression when I get home, I don't want to install 500MB active python to my work PC just to test this.

**Comment by rpl** [comment]
@icl7126 don't worry I did run a mozregression and it pointed to the following pushlog:

https://hg.mozilla.org/mozilla-central/pushloghtml?fromchange=546d1fd47c9a48b6919848b2c6f28359460731eb&tochange=73353f375d1dda3d4ff434da2a4f7bdfd39d342

Looking at the changes part of that pushlog, the regression should have been introduced recently by [Bug 1522571](https://bugzilla.mozilla.org/show_bug.cgi?id=1522571) (and it seems to be also confirmed by another regression bug linked to that issue: [Bug 1578108](https://bugzilla.mozilla.org/show_bug.cgi?id=1578108), a vscode regression related to "installing an extension temporarily using the remote debugging actors", which is also what the web-ext run error is related to).

**Comment by icl7126** [comment]
Thanks!
Feel free to close this if it's gonna be fixed by FF team.

**Comment by Rob--W** [comment]
Fixed in Firefox DevEd 70.0b4.

**Comment by BradSharp** [comment]
I am also experiencing this with Firefox v112 when using `web-ext run` with a copy of the multi-account-containers extension.
